dcicutils/misc_utils.py

```diff
@@ -325,3 +325,100 @@ def apply_dict_overrides(dictionary: dict, **overrides) -> dict:
 
 def utc_today_str():
     return datetime.datetime.strftime(datetime.datetime.utcnow(), "%Y-%m-%d")
+
+
+class LockoutManager:
```

Comment:
It would be nice to have some more documentation here on how this works/is used.

Reply:
Doc added.